why-apollo-advantages-disadvantages-alternatives.md

```diff
@@ -51,93 +51,90 @@ There are plenty of popular and tech savvy companies out there who are using Apo
 
-One of Apollo's libraries makes it possible to connect Apollo Client to React. In the same fashion as other libraries (Redux, MobX, ...) did it before, the react-apollo library offers higher-order components (and render prop components) to connect both worlds. However, there are other libraries out there bridging not only Apollo Client to React, but also Apollo to Angular or Apollo to Vue. That's what makes Apollo Client view layer agnostic which is great for the growing JavaScript ecosystem.
+One of Apollo's libraries makes it possible to connect Apollo Client to React. Just like libraries like Redux and MobX, the React-aApollo library has higher-order and render prop components to connect both worlds. However, there are other libraries out there that bridge not only Apollo Client to React, but also Apollo to Angular or Apollo to Vue. That's what makes Apollo Client view layer agnostic, which is great for the growing JavaScript ecosystem.
 
-Apollo is not only on the client-side library agnostic. Also on the server-side for Node.js it offers several solutions to connect to different Node.js libraries. Whereas Apollo Server for Express.js is the most popular choice among developers and companies, there are other solutions for Koa and Hapi on Node.js for Apollo Server as well.
+Apollo is also library agnostic on the server-side, and it offers several solutions to connect with Node.js libraries. Apollo Server for Express.js one of the most popular choices among developers and companies, and there are other solutions for Koa and Hapi on Node.js for Apollo Server as well.
 
 {{% sub_chapter_header "Modern Data Handling with Apollo" "apollo-data-handling" %}}
 
-Do you remember the cases where you had to trigger data fetching in one of your component's lifecycle methods in an imperative way? These times are over with Apollo Client, because you query your data in a declarative way with Apollo Client. Most often you will use a higher-order component or render prop in React which triggers a query automatically when rendering the React component. Only the GraphQL mutations are triggered imperatively, but only because a higher-order component or render prop gives you access to the function which executes the mutation (e.g. on a button click). Apollo embraces declarative programming over imperative programming.
+Remember back when we had to trigger data fetching in one a component's lifecycle methods imperatively? Apollo Client solves this, because its data queries are declarative. React often employs a higher-order component or render prop to trigger a query automatically when a component renders. The GraphQL mutations are triggered imperatively, but that's only because a higher-order component or render prop grants access to the function which executes the mutation (e.g. on a button click). Essentially, Apollo embraces declarative programming over imperative programming.
```

Comment:
"in one a component's lifecycle methods imperatively?" -> "in a component's lifecycle methods imperatively?"